src/autocomplete/src/Autocomplete.js

```diff
@@ -163,10 +168,7 @@ export default class Autocomplete extends PureComponent {
                   index,
                   style,
                   children: itemString,
-                  onMouseUp: () => {
-                    selectItemAtIndex(index)
-                  },
-                  isSelected: itemToString(selectedItem) === itemString,
+                  isSelected: selectedItem === item,
```

Comment:
This avoids an unnecessary string conversion.

Reply:
This is how they do in the Downshift docs though? https://github.com/paypal/downshift#usage

Reply:
I think you're right, I'll revert this change.